scripts/upload-cli-binaries.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eo pipefail
+IFS=$'\n\t'
+
+for os in "linux" "darwin" "windows"
+do
+    sha256sum bazel-bin/infrastructure/cli/airy_"$os"_bin > bazel-bin/infrastructure/cli/airy_"$os"_bin.sha
```

Comment:
I would recommend the following filename: `airy_"$os"_sha256sum.txt`

Reply:
right that is also how kubectl does it. fixed.